frontend/apps/webapp/src/core/components/Branding/BrandingProvider.tsx

```diff
@@ -1,9 +1,9 @@
-import axios, { AxiosResponse } from 'axios';
+// import axios, { AxiosResponse } from 'axios';
 import React, { FC, Fragment, useEffect, useMemo, useState } from 'react';
 import { useDispatch, useSelector } from 'react-redux';
 import { useLocation } from 'react-router-dom';
 import { StoreState } from '../../../types';
-import { StyleMatchType } from './brandingService';
+// import { StyleMatchType } from './brandingService';
```

Comment:
Same for all the other lines we have commented out.  
Let's delete them to clean up the source.  
If we need to see the history, it's available in Git